app/js/services/filter.service.js

```diff
@@ -630,7 +630,7 @@ function FilterService(
         }
 
         //age
-        if (includeParam('age') && _age !== '0') {
+        if (includeParam('age') && _age !== 0) {
```

Comment:
selecting 'All' for the Age filter sets the age property to '0' which results in 0 results since _age !== 0.

Reply:
This could also be fixed by deleting the url age param when it is set to 0 on rules page load